llvm/tools/llvm-objcopy/ObjcopyOpts.td

```diff
@@ -176,6 +176,9 @@ def G : JoinedOrSeparate<["-"], "G">,
         Alias<keep_global_symbol>,
         HelpText<"Alias for --keep-global-symbol">;
 
+def no_verify_note_sections : Flag<["--"], "no-verify-note-sections">,
```

Comment:
We should have both positive and negative versions of this option, and you need to show that:
1) The default behaviour is the "positive" version (i.e. show the behaviour with and without the positive option only is the same).
2) The negative option suppresses the behaviour when no positive option is specified.
3) If both are specified, the last one wins (this will require at least two separate invocations, one for each option being last).

Please also review and decide whether the new option(s) should be added to the list of options that are warned about when processing files of non-ELF formats (see ConfigManager).